installation.rst

```diff
@@ -198,9 +198,16 @@ Installation on MS Windows
 
 We are recommending and supporting installation of QuTiP into a Conda environment. Other scientific Python implementations such as Python-xy may also work, but are not supported.  
 
-As of QuTiP 4.1, recommended installation on Windows requires Python 3.5+, as well as Visual Studio 2015.  With this configuration, one can install QuTiP using any of the above mentioned receipes. Visual Studio 2015 is not required for the install of the conda-forge package, but it is required at runtime for the string format time-dependence solvers. When installing Visual Studio 2015 be sure to select options for the C++ compiler.
+As of QuTiP 4.1, recommended installation on Windows requires Python 3.5+, as well as MS Visual C++ Build Tools. MS Visual C++ Build Tools are strictly required for the install of the conda-forge package, but MS Visual Studio 2015 (or more recent) is recommended because it is required at runtime for the string format time-dependence solvers. When installing MS Visual Studio be sure to select the following components: 
```

Comment:
> Build Tools are strictly required for the install of the conda-forge package, but MS Visual Studio 2015 (or more recent) is recommended because it is required at runtime for the string format time-dependence solvers.

Do you know if there is any problem if one only install the build tools? I'm also using build tools and haven't experienced any problem so far. I'm wondering if we can say that VS C++ build tools are enough instead of recommendingVS2015.

Reply:
Long answer short, VS C++ build tools are enough, that is correct.

However, as I said in the lines 206 and following, using the developer command prompt saves time in adjusting the PATH variables a few times, to ensure Windows locates all the dependencies downloaded with the build tools. 
As far as I could see that comes with VS2015+, but maybe there is a shortcut to install it otherwise? 

Again, an expert user can circumvent the issue with little patience. VS2015 saves time trading off for space :D